tests/integrationv2/test_well_known_endpoints.py

```diff
@@ -0,0 +1,65 @@
+import copy
+import os
+import pytest
+
+from configuration import available_ports, PROTOCOLS
+from common import ProviderOptions, Protocols, Ciphers
+from fixtures import managed_process
+from providers import Provider, S2N
+from utils import invalid_test_parameters, get_parameter_name
+
+
+ENDPOINTS = [
+    {"endpoint": "amazon.com"},
+    {"endpoint": "facebook.com"},
+    {"endpoint": "google.com"},
+    {"endpoint": "netflix.com"},
+    {"endpoint": "s3.amazonaws.com"},
+    {"endpoint": "twitter.com"},
+    {"endpoint": "wikipedia.org"},
+    {"endpoint": "yahoo.com"},
+]
+
+
+if os.getenv("S2N_NO_PQ") is None:
```

Comment:
aside: This flag name is unfortunate and it makes this line read poorly

Reply:
It is fairly annoying. We are adding support to s2n to tell us more information about compilation options in #1977. Hopefully we'll return whether PQ is supported instead of whether PQ is not not supported.

The reason this works here is that this env var matches the name of the define in the code, and in the build system.